packages/teleport/src/services/joinToken/types.ts

```diff
@@ -20,3 +20,10 @@ export type JoinToken = {
 };
 
 export type Roles = 'App' | 'Node' | 'Db';
+
+export type Method = 'token' | 'ec2' | 'iam';
+
+export type Rule = {
+  awsAccountId?: string;
```

Comment:
maybe? b/c it's required now?

```suggestion
  awsAccountId: string;
```